### DISCUSSION TITLE
EDatepicker: Wrong Input to Type "is-danger"

### DISCUSSION THREAD

**Discussion by julzELO** [buefy/buefy#3997] [Ideas]
### Description

Hello, so first of all, im not sure if its already possible, if not I think it would be a good feature for the Datepicker.

If an input is wrong and the field is left, the field will be emptied automatically. Here an extension should be introduced, so that a flag can be set that the field is not emptied but becomes of the type "is-danger".

### Why Buefy need this feature

If the user has only one typo, the field will be cleared, in this case it would be better for the User that the value remains but the field becomes invalid.

**Comment by jtommy** [comment]
Mmm I should think about it, you are right but the datepicker has to emit a new value (null)
